src/extension.ts

```diff
@@ -263,8 +263,8 @@ export async function activate(context: vscode.ExtensionContext) {
     }
 
     ext.s3fileViewerManager = new S3FileViewerManager()
+    await ext.s3fileViewerManager.createTemp()
```

Comment:
these 2 lines should be near the top as much as possible. e.g. around line 122 . definitely not outside of the try/catch.